### PR TITLE
Add hints for Servo debugging

### PR DESCRIPTION
**Pull request by AndyZe**
Add some hints for questions we get over and over again. Also for some "active learning"  ;)

### CODE REVIEW

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 1]
```diff
@@ -51,6 +51,23 @@ Expected Output
 
 Note that the controller overlay here is just for demonstration purposes and is not actually included
 
+Introspection
+-------------
+
+Here are some tips for inspecting and/or debugging the system.
+
+View the ros2_controllers that are currently active with :code:`ros2 control list_controllers`. You will see a JointTrajectoryController that receives the joint position commands from Servo and handles them in the simulated robot driver. Servo is also compatible with JointGroupPosition or JointGroupVelocity-type controllers.
```
[the remark this answers is not in the document]
Comment (mlautman) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:59]: You might also consider adding `:moveit_codedir:` links to JointTrajectoryController, JointGroupPosition and JointGroupVelocity header files in the moveit2 codebase

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 2]
```diff
@@ -51,6 +51,23 @@ Expected Output
...
+
+:code:`ros2 topic echo /servo_node/status` shows the current state of the Servo node. If :code:`0` is published, all is well. The definition for all enums can be seen :moveit_codedir:`here.<moveit_ros/moveit_servo/include/moveit_servo/status_codes.h>`
+
+:code:`ros2 node list` shows the following. :code:`ros2 node info` can be used to get more information about any of these nodes.
+
+- :code:`joy_node` handles commands from the XBox controller
```
Comment (mlautman) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: Maybe make all of the tips bullet points or none of them bullet points

**Review by mlautman — COMMENTED** [review]
Some quick feedback. Note, I don't have a linux machine at the moment so I can't actually build the tutorials locally to verify formatting etc..

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst (hunk 2 above)
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: Thanks @mlautman, hope life is good

Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]: and shame on you for not having a Linux machine

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 3]
```diff
@@ -51,6 +51,23 @@ Expected Output
 
 Note that the controller overlay here is just for demonstration purposes and is not actually included
 
+Introspection
+-------------
+
+Here are some tips for inspecting and/or debugging the system.
+
+#. View the :code:`ros2_controllers` that are currently active with :code:`ros2 control list_controllers`. You will see a `JointTrajectoryController <https://github.com/ros-controls/ros2_controllers/tree/master/joint_trajectory_controller>`_ that receives the joint position commands from Servo and handles them in the simulated robot driver. The JointTrajectoryController is very flexible; it can handle any combination of position/velocity/(position-and-velocity) input. Servo is also compatible with `JointGroupPosition <https://github.com/ros-controls/ros2_controllers/tree/master/position_controllers>`_ or `JointGroupVelocity <https://github.com/ros-controls/ros2_controllers/tree/master/velocity_controllers>`_-type controllers.
```
Comment (corycrean) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:59]: The command `ros2 control list controllers` requires `ros2_control` to be installed.  Can you add a note that mentions this, or possibly add it as a dependency in `package.xml`?

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 4]
```diff
@@ -51,6 +51,23 @@ Expected Output
...
+
+#. :code:`ros2 topic echo /servo_node/status` shows the current state of the Servo node. If :code:`0` is published, all is well. The definition for all enums can be seen :moveit_codedir:`here.<moveit_ros/moveit_servo/include/moveit_servo/status_codes.h>`
+
+#. :code:`ros2 node list` shows the following. :code:`ros2 node info` can be used to get more information about any of these nodes.
+
+   - :code:`joy_node` handles commands from the XBox controller
```
Comment (corycrean) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:65]:
```suggestion
   - :code:`/joy_node` handles commands from the XBox controller
```

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 5]
```diff
@@ -51,6 +51,23 @@ Expected Output
...
+
+   - :code:`moveit_servo_demo_container` holds several ancillary ROS2 "component nodes" that are placed in a container for faster intra-process communication
```
Comment (corycrean) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:67]:
```suggestion
   - :code:`/moveit_servo_demo_container` holds several ancillary ROS2 "component nodes" that are placed in a container for faster intra-process communication
```

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst [hunk 6]
```diff
@@ -51,6 +51,23 @@ Expected Output
...
+
+   - :code:`servo_node` which does the calculations and collision checking for this demo. :code:`servo_node` may be moved into the demo container in the future
```
Comment (corycrean) [inline doc/examples/realtime_servo/realtime_servo_tutorial.rst:69]:
```suggestion
   - :code:`/servo_node` which does the calculations and collision checking for this demo. :code:`servo_node` may be moved into the demo container in the future
```

**Review by corycrean — CHANGES_REQUESTED** [review]

File: doc/examples/realtime_servo/realtime_servo_tutorial.rst (hunk 3 above)
Comment (AndyZe) [reply doc/examples/realtime_servo/realtime_servo_tutorial.rst:59]: good point, done

**Review by corycrean — APPROVED** [review]

**Review by tylerjw — APPROVED** [review]

**Pull request merged**